ci/environment-dashboard.yml

```diff
@@ -4,8 +4,8 @@ dependencies:
   - python =3.10
   - sqlalchemy =2.0
   - altair =5.0
-  - bokeh =2.4
-  - panel =0.14
+  - bokeh =3.2
+  - panel =1.2
```

Comment:
Duplicates https://github.com/coiled/benchmarks/pull/983, which has issues. Please revert here.

Reply:
https://github.com/coiled/benchmarks/pull/1061/commits/ea18dfe04cc941950622e72b5e2600c3ccccd073 seems to fix links failing to render, based on reading at [their change in default renderer since v1](https://panel.holoviz.org/reference/panes/Markdown.html#renderer) I think it (markdown-it) was interpreting the `.` incorrectly within the Python version.

Reply:
Moved to #983